src/wrappers/PhylorefWrapper.js

```diff
@@ -287,6 +290,30 @@ class PhylorefWrapper {
     });
   }
 
+  /**
+   * Returns the nomenclatural code used by this phyloref.
+   *
+   * If all of the specifiers are taxon concepts with the same nomenclatural code
+   * (using the default nomenclatural code set during constructions of this object
+   * when one isn't provided), this will return that nomenclatural code. Otherwise,
+   * this method will return owlterms.UNKNOWN_CODE.
+   */
+  get nomenCode() {
```

Comment:
I think the summarized getter is useful, but I agree that the list of unique nomenclatural codes might be useful to users as well. I've therefore split that off into a separate getter named `uniqNomenCodes`, while you can get the summarized nomenclatural code with the getter `summarizedNomenCode`. This is in 6f0f880c146.

Reply:
Not sure how that's "summarized" given what it returns? I.e., isn't that adding more confusion rather than reducing?

Reply:
How about `soleNomenCode` or `onlyNomenCode`? We could also go back to just `nomenCode` (in addition to `uniqNomenCodes`) -- that might be cleanest?

Reply:
Why not `defaultNomenCode`? Isn't that what the semantics are?

Reply:
I'd argue that `defaultNomenCode` should be used when the nomenclatural code is used as a default for taxon concepts that don't explicitly set their nomenclatural code (there is actually a Phyx-level property that does that, `defaultNomenclaturalCodeURI`, that I've just realized wasn't in the Phyx manuscript, so I added it there). This getter is a summary of the nomenclatural codes present in the specifiers of this phyloreference, using the `defaultNomenCode` in cases where a taxon concept specifier doesn't have an explicit nomen code set.

Note that one difference between these two concepts is that a summarized nomenclatural code only makes sense when the phyloreference is made up of one or more specifiers that *can* have nomenclatural codes (which, right now, is only taxon concept specifiers). If there is at least one specifier that can't have a nomenclatural code (such as a specimen), then the summarized nomenclatural code should be undefined, but I would expect a `defaultNomenCode` to continue to reflect the Phyx-level default in that case.

Reply:
> If there is at least one specifier that can't have a nomenclatural code (such as a specimen), then the summarized nomenclatural code should be undefined

Why? I wouldn't think so.

Reply:
What should it be?

Reply:
If there's a specifier that can't have a nomenclatural code, why should it affect the determination of whether there's a single nomenclatural code that can or should be used as default?

Reply:
There are two issues here that have gotten entangled:
1. What should we call the method that returns the "default" nomen code. While I continue to disagree, I've gone ahead and renamed that getter as `defaultNomenCode`. The previous `defaultNomenCode` has now been renamed to `phyxDefaultNomenCode`. I've made this change in e078ba7d5b5 and added tests for this in 571b6e076a0.
2. I've modified the `defaultNomenCode` getter so that specifiers without a nomenclatural code are ignored when determining the default nomenclatural code. I've made this change in 61ddffce589.

Let me know if this works for you!